--- gnome-online-accounts-3.26.2/src/goabackend/goaowncloudprovider.c~	2017-12-14 17:45:39.000000000 +0100
+++ gnome-online-accounts-3.26.2/src/goabackend/goaowncloudprovider.c	2018-12-26 09:12:36.749051768 +0100
@@ -243,7 +243,7 @@
 
  out:
   g_clear_object (&password_based);
-  g_clear_pointer (&uri, (GDestroyNotify *) soup_uri_free);
+  g_clear_pointer (&uri, (GDestroyNotify) soup_uri_free);
   g_free (uri_string);
   return ret;
 }
@@ -446,7 +446,7 @@
   ret = soup_uri_to_string (uri, FALSE);
 
  out:
-  g_clear_pointer (&uri, (GDestroyNotify *) soup_uri_free);
+  g_clear_pointer (&uri, (GDestroyNotify) soup_uri_free);
   g_free (scheme);
   g_free (uri_string);
   return ret;
